src/cr-logging.eventstore/EventStoreSerilogLogger.cs

```diff
@@ -0,0 +1,69 @@
+﻿// <copyright file="EventStoreSerilogLogger.cs" company="Cognisant">
+// Copyright (c) Cognisant. All rights reserved.
+// </copyright>
+
+namespace CR.Logging.EventStore
+{
+    using System;
+    using global::EventStore.ClientAPI;
+    using Serilog.Core;
+
+    /// <summary>
+    /// Extension to support logging of EventStore events.
+    /// </summary>
+    public class EventStoreSerilogLogger : ILogger
+    {
+        private readonly Logger _logger;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EventStoreSerilogLogger"/> class.
+        /// </summary>
+        /// <param name="logger">Serilog Logger</param>
+        public EventStoreSerilogLogger(Logger logger) => _logger = logger;
+
+        /// <summary>
+        /// Log Error
```

Comment:
We should probably put better logs here, given it's a wrapper around the Serilog `Logger` class their docs should provide some inspiration.